dask_cuda/benchmarks/utils.py

```diff
@@ -185,6 +185,18 @@ def parse_benchmark_args(description="Generic dask-cuda Benchmark", args_list=[]
         "Note: --devs is currently ignored in multi-node mode and for each host "
         "one worker per GPU will be launched.",
     )
+    parser.add_argument(
+        "--show-p2p-bandwidth",
+        action="store_true",
+        help="Produce detailed point to point bandwidth stats in output",
+        default=True,
+    )
```

Comment:
```suggestion
```

Reply:
We don't need this, just the `--no-...` option should suffice to assume the enabled option is the default (e.g., https://github.com/rapidsai/dask-cuda/blob/86a9846b1b8f1bea7bdc37c1df09a75f981f1fbf/dask_cuda/benchmarks/utils.py#L193-L197 ).